project-koku/koku
# 1018 reject invalid queries

reference #1018 

Testing:

1. Go to '/api/cost-management/v1/costmodels/'
2. add invalid query /api/cost-management/v1/costmodels/?wrong=query
3. See returned data
```python
HTTP 400 Bad Request
Allow: GET, POST, HEAD, OPTIONS
Content-Type: application/json
Vary: Accept

{
    "errors": [
        {
            "detail": "Cannot resolve keyword 'wrong' into field. Choices are: costmodelmap, created_timestamp, description, name, rates, source_type, updated_timestamp, uuid",
            "source": "detail",
            "status": 400
        }
    ]
}
```
